Mathlib/RingTheory/LocalRing/MaximalIdeal/Basic.lean

```diff
@@ -128,3 +128,10 @@ end IsLocalRing
 
 theorem IsLocalRing.maximalIdeal_eq_bot {R : Type*} [Field R] : IsLocalRing.maximalIdeal R = ⊥ :=
   IsLocalRing.isField_iff_maximalIdeal_eq.mp (Field.toIsField R)
+
+lemma Subring.isLocalRing_of_unit {R : Type*} [CommRing R] [IsLocalRing R] (S : Subring R)
```

Comment:
```suggestion
lemma Subring.isLocalRing_of_unit {R : Type*} [Ring R] [IsLocalRing R] (S : Subring R)
```

It also works with `[Semiring R]` and `(S : Subsemiring R)`, but that might be out of the scope.